README.md

```diff
@@ -1 +1,49 @@
-# ES-Awesome-book
\ No newline at end of file
+# ES-Awesome-book
+
+ES6-Awesome-book
+In this project I made a website that stores books in a collection and dynamcally displays them at the top of the page.
+
+Built With
+HTML
+CSS
+JavaScript
+
+Live Demo (if available)
+Live Demo Link
+
+Getting Started
+To get a local copy up and running follow these simple example steps.
+Open your console.
+Type the following "git clone https://github.com/ijeomaodinko/Es-Awesome-book.git" if you are using HTTPS or "git clone git@github.com:ijeomaodinko/E6-Awesome-book.git" if you are using a SSH key.
+Press ENTER to create your local clone.
+After creating it, you can open the "index.html" file in your browser.
+
+Prerequisites
+Setup
+Install
+Usage
+Run tests
+Deployment
+
+Author
+
+Odinko Ijeoma
+
+GitHub: @ijeomaodinko
+Twitter: @iodinko
+LinkedIn: @ijeomaodinko
+
+handshake Contributing
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the issues page.
+
+Show your support
+Give a star if you like this project!
+
+Acknowledgments
+Hat tip to anyone whose code was used
+Inspiration
+etc
+memo License
+This project is MIT licensed.
```

Comment:
- [ ] **[OPTIONAL]** As recommended by the previous reviewer, I think you should consider formatting your README to be better. You can look at the raw version of the Microverse template here to better understand how to write more organized markdown file.

  [LINK TO RAW README](https://raw.githubusercontent.com/microverseinc/readme-template/master/README.md)